packages/react-vapor/src/components/browserPreview/BrowserPreview.tsx

```diff
@@ -6,19 +6,23 @@ import {Tooltip} from '../tooltip';
 
 export interface BrowserPreviewProps {
     headerDescription?: string;
+    title?: string;
 }
 
-export const BrowserPreview: React.FunctionComponent<BrowserPreviewProps> = ({children, headerDescription}) => (
+export const BrowserPreview: React.FunctionComponent<BrowserPreviewProps> = ({children, title, headerDescription}) => (
```

Comment:
`children, headerDescription, title}`

I prefer alphabetical order for the maintenance.